docs/updating/8-0.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Updating to v8
+---
+
+# Updating from Ionic 7 to 8
+
+:::note
+This guide assumes that you have already updated your app to the latest version of Ionic 7. Make sure you have followed the [Upgrading to Ionic 7 Guide](./7-0) before starting this guide.
+:::
+
+:::info Breaking Changes
+For a **complete list of breaking changes** from Ionic 7 to Ionic 8, please refer to [the breaking changes document](https://github.com/ionic-team/ionic-framework/blob/feature-8.0/BREAKING.md#version-8x) in the Ionic Framework repository.
```

Comment:
Is there a Jira ticket to track that this link will need to be updated when it's released?

Reply:
Yes, this is being tracked in FW-5941.